.github/workflows/ci.yml

```diff
@@ -82,9 +82,6 @@ jobs:
           cache: 'yarn'
           cache-dependency-path: ./packages/sdks/yarn.lock
 
-      - name: Remove mitosis resolutions from package.json
-        run: ./scripts/remove-mitosis-resolutions.sh
```

Comment:
Ended up removing this from CI, because at the end of the day, the developer has to remember to undo the sym-link or it'll get checked into `yarn.lock`. No way to avoid having developers adding/removing symlinks before/after comitting